app/controllers/api/projects/remixes_controller.rb

```diff
@@ -14,6 +14,7 @@ def index
 
       def show
         @project = Project.find_by!(remixed_from_id: project.id, user_id: current_user&.id)
+        authorize! :show, @project
```

Comment:
just a thought, we're using cancancan a bit inconsistently in places, and I wonder if `load_and_authorize_resource :project, only: [:show]` could be a cleaner approach?

Reply:
would that still work with the way we're loading up the remix project?